### ISSUE TITLE
Add Star Aligner

### ISSUE THREAD

**Issue by bentsherman** [SystemsGenetics/GEMmaker#133]
Reed found a paper where the authors combined GTEx and TCGA fastq files into a single GEM. Paper is here:

https://www.nature.com/articles/sdata201861

They used the Star aligner, which is apparently pretty popular, and a few other tools that GEMmaker doesn't include. I figured it might be useful for GEMmaker to provide these tools in addition to salmon and kallisto.

**Comment by spficklin** [comment]
Good idea! I'm marking as low-priority for now until we can get some of the bigger issues fixed.
